docs/_releases/v2.0.0-pre.4/matchers.md

```diff
@@ -112,6 +112,26 @@ Requires the value to be a `Function`.
 Requires the value to be an `Array`.
 
 
+#### `sinon.match.array.deepEquals(iterable)`
+
+Requires an iterable value to be deep equal another one.
```

Comment:
Once you see it written down, this is weird. You're checking inalterable from the array matcher. I would expect the array namespace to contain array matchers.